### PR TITLE
Support WRED profiles on system ports 

### PR DESCRIPTION
**Pull request by vmittal-msft**
**What I did**
Added support to apply wred profile on system ports

**Why I did it**
remote system ports were missing wred profile

**How I verified it**
Tested on sonic chassis

**Details if related**

### CODE REVIEW

**Comment by vmittal-msft** [comment]
https://github.com/sonic-net/sonic-swss/pull/2832

File: orchagent/bufferorch.cpp [hunk 1]
```diff
@@ -1001,9 +1018,19 @@ task_process_status BufferOrch::processQueue(KeyOpFieldsValuesTuple &tuple)
...
-            if (gPortsOrch->isPortAdminUp(port_name)) {
-                SWSS_LOG_WARN("Queue profile '%s' applied after port %s is up", key.c_str(), port_name.c_str());
+            if(local_port == true)
+            {
+                if (gPortsOrch->isPortAdminUp(local_port_name)) {
+                    SWSS_LOG_WARN("Queue profile '%s' applied after port %s is up", key.c_str(), port_name.c_str());
+                }
+            }
+            else
+            {
```
Comment (kenneth-arista) [inline orchagent/bufferorch.cpp:1028]: Looks like the difference here is the param passed to `isPortAdminUp`. Can you change it to the following? 

```
            auto &tmp_port_name = local_port ? local_port_name : port_name;
            if (gPortsOrch->isPortAdminUp(tmp_port_name)) {
                SWSS_LOG_WARN("Queue profile '%s' applied after port %s is up", key.c_str(), port_name.c_str());
            }

```

**Comment by kenneth-arista** [comment]
portsorch.cpp change looks fine. I don't have enough background to review the other changes.

**Comment by vmittal-msft** [comment]
Opened following PR to take care of this issue  - 
https://github.com/sonic-net/sonic-swss/pull/2855

**Pull request closed without merging**